crates/ide_completion/src/item.rs

```diff
@@ -144,6 +144,21 @@ pub struct CompletionRelevance {
     /// }
     /// ```
     pub exact_type_match: bool,
+    /// This is set in cases like these:
+    ///
+    /// ```
+    /// fn foo(bar: u32) {
+    ///     $0 // `bar` is local
+    /// }
+    /// ```
+    ///
+    /// ```
+    /// fn foo() {
+    ///     let bar = 0;
+    ///     $0 // `bar` is local
+    /// }
+    /// ```
```

Comment:
Seems like we can condense both examples to one, to reduce the volume of comments without compromising on the amount on knowledge they convey.

Reply:
Agreed! Since this has merged I'll try to remember to pick this up in a future PR.